src/styles/main.scss

```diff
@@ -1 +1,47 @@
-// put code here
+@keyframes rotate-minute-hand {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+@keyframes rotate-hour-hand {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
```

Comment:
this animations are the same, you can just have 1 animation and use it both places